kani-driver/src/cbmc_output_parser.rs

```diff
@@ -285,7 +298,7 @@ pub struct TraceValue {
     pub name: String,
     pub binary: Option<String>,
     pub data: Option<TraceData>,
-    pub width: Option<u8>,
+    pub width: Option<u32>,
```

Comment:
Could we stick a small unit test in this file with a reduce version of the string we saw? It should be possible to just deserialize a string to a `TraceValue` to unit test just this code. It might help when we make changes to this in the future to accumulate a list of corner cases we ran into.

Reply:
Yes, let me do that.

Reply:
Done.